### DISCUSSION TITLE
Set current Time for Timepicker

### DISCUSSION THREAD

**Discussion by Torchok19081986** [MahApps/MahApps.Metro#4506] [Q&A] [answered]
Hallo , i would to use timepicker, but here i dont know, how to set current time to timepicker control. SourceHours or SourceMinutes is IEnumenurable<int> and not the value. Any idea ?

**Comment by timunie** [comment]
@Torchok19081986 why do you file it as a bug? This is a question!!!

There is a property SelectesValue or SelectedDateTime, don't remember the exact name. The hours and minutes sour es are for the drop-down for you to configure.

**Comment by Torchok19081986** [comment]
moin,  @timunie. Many thanks for answer. Unfortunatelly, i lost all account data and key for gitter. I thought i can make a question here.  
Found it, this , just like you said, Property is SelectedDateTime. But this property got me date and time. I need separate date and time in datepicker from mahapps for 2 controls: one for date and one for time. Standard datepicker from wpf is not really good for this purpose.

**Reply by timunie** [reply, marked as answer]
You can ask here. Gitter is kinda dead anyway afaik. BUT use Discussions tab and open a Q&A instead of a bug report. That's my point.

**Reply by Torchok19081986** [reply]
thanks again. any idea , how to separate date and time in mahapps datepicker and use it one control as date and second one as time ?
